flipperdevices/flipperzero-firmware
# [FL-3608] Fix iButton crash on missing file

# What's new

- Fix crash when attempting to emulate a nonexistent file in iButton.

# Verification 

1. Create any iButton key
2. Share the key via the Android app (choose Export File)
3. Import it back to the app, wait until it syncs
4. Emulate it - Flipper should not crash.

# Checklist (For Reviewer)
